### DISCUSSION TITLE
Inserters dont unload the train.

### DISCUSSION THREAD

**Discussion by Georgekorde** [Opinionated-Blueprints/10-Books-Full-of-Rails#95] [Q&A] [answered]
Hello, im not very good with train networks and logistics...i cant seem to figure out why the inserters dont work...im using the cityblock blueprint and i used the blueprint from the requester book to unload the train but the inserters dont work. Could you explain what am i doing wrong and how to fix it please ? 

![Screenshot 2024-11-29 155120](https://github.com/user-attachments/assets/5a9752b6-7204-43a1-bbc4-eec7e5e64591)

Thank you.

**Comment by Bocian-1** [comment]
you didn't pick the item when placing the station
The item you pick sets the filter on the filter inserters so that they don't unload the wrong items if a train with mixed cargo arrives
https://github.com/Opinionated-Blueprints/10-Books-Full-of-Rails/wiki/Provider-and-Requester-stations

**Reply by Frerduro** [reply]
I think I see the issue actually. It looks like he didn't connect the red & green wire between the station section and the unloader section

**Reply by Georgekorde** [reply]
So funny thing insted of putting the train station blueprint of the city blocks( the left one ) i used the one you made in the requester book and it worked fine with the filter. any idea why ?

![Screenshot 2024-11-29 165930](https://github.com/user-attachments/assets/94f86cd9-2eed-46ce-9694-fcc521eb68f1)

**Reply by Bocian-1** [reply, marked as answer]
...
![image](https://github.com/user-attachments/assets/7fd7b12b-add2-40ff-b6dd-7cb049970154)
Work in progress on the next update so there are more combinators on the requesters, but it should look like this, though you can have the belts go whatever direction
![image](https://github.com/user-attachments/assets/3e439a5f-24e7-4288-9887-02a0e28782d6)
